contrib/cirrus/setup_environment.sh

```diff
@@ -304,6 +304,14 @@ case "$TEST_FLAVOR" in
             die "Invalid value for \$TEST_ENVIRON=$TEST_ENVIRON"
         fi
 
+        install_test_configs
+        ;;
+    minikube)
+        dnf install -y $PACKAGE_DOWNLOAD_DIR/minikube-latest*
+        remove_packaged_podman_files
+        make install.tools
+        make install PREFIX=/usr ETCDIR=/etc
+        minikube config set driver podman
```

Comment:
Do you know if the above command will actually run podman in any way?  If so, you may want to move it down a line (below `install_test_configs`).

Reply:
it won't run podman, it just configures minikube to use podman when we start it.